test/screenshot/temporary-package/temporary-package.scss

```diff
@@ -3,4 +3,6 @@
 
 body {
   color: blue;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
never seen moz-osx 👍

Reply:
Stole this from [here](https://github.com/material-components/material-components-web/blob/master/packages/mdc-typography/_mixins.scss#L19-L23)